cfme/tests/automate/test_method.py

```diff
@@ -264,3 +265,90 @@ def test_task_id_for_method_automation_log(request, generic_catalog_item):
     wait_for(lambda: service_request.request_state == "active", fail_func=service_request.reload,
              timeout=60, delay=3)
     result.validate_logs()
+
+
+@pytest.mark.meta(blockers=[BZ(1704439)])
+@pytest.mark.meta(server_roles="+notifier")
+def test_send_email_method(request, klass):
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/20h
+        caseimportance: high
+        testtype: functional
+        startsin: 5.10
+        casecomponent: Automate
+
+    Bugzilla:
+        1688500
+        1702304
+    """
+    mail_to = fauxfactory.gen_email()
+    mail_cc = fauxfactory.gen_email()
+    mail_bcc = fauxfactory.gen_email()
+
+    # Ruby code to send emails
+    script = (
+        'to = "{mail_to}"\n'
+        'subject = "Hello"\n'
+        'body = "Hi"\n'
+        'bcc = "{mail_bcc}"\n'
+        'cc = "{mail_cc}"\n'
+        'content_type = "message"\n'
+        'from = "cfadmin@cfserver.com"\n'
+        "$evm.execute(:send_email, to, from, subject, body, {{:bcc => bcc, :cc => cc,"
+        ":content_type => content_type}})"
+    )
+    script = script.format(mail_cc=mail_cc, mail_bcc=mail_bcc, mail_to=mail_to)
+
+    # Adding schema for executing method - send_email which helps to send emails
+    klass.schema.add_fields({'name': 'execute', 'type': 'Method', 'data_type': 'String'})
+
+    # Adding method - send_email for sending mails
+    method = klass.methods.create(
+        name=fauxfactory.gen_alphanumeric(),
+        display_name=fauxfactory.gen_alphanumeric(),
+        location='inline',
+        script=script)
+
+    # Adding instance to call automate method - send_email
+    instance = klass.instances.create(
+        name=fauxfactory.gen_alphanumeric(),
+        display_name=fauxfactory.gen_alphanumeric(),
+        description=fauxfactory.gen_alphanumeric(),
+        fields={'execute': {'value': method.name}}
+    )
+
+    # Setting Outgoing SMTP E-mail Server
+    klass.appliance.server.settings.update_smtp_server(
```

Comment:
You can directly use the `smtp_test` fixture instead of this.
Refer this: https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/configure/test_email.py#L7